src/utils/general.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Checks whether an object is instance of a given class
+ * @param {Object} instance  the instance object to check
+ * @param klass     the class the instance object to be checked against
+ * @returns {boolean} whether the instance is instanceof the provided klass
+ */
+module.exports.instanceOf = (instance, klass) => {
```

Comment:
Can this be moved to here? https://github.com/twilio/twilio-cli-core/blob/master/src/services/javascript-utilities.js